packages/vkui/src/components/Snackbar/subcomponents/Basic/Basic.tsx

```diff
@@ -61,6 +67,7 @@ export function Basic({
   children,
   ...restProps
 }: SnackbarBasicProps) {
+  const { sizeY } = useAdaptivity();
```

Comment:
```suggestion
  const { sizeY = 'none' } = useAdaptivity();
```

по умолчанию приходит `undefined`